sdk/lib/opentelemetry/sdk/correlation_context/builder.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module CorrelationContext
+      # No op implementation of CorrelationContext::Builder
+      class Builder
+        attr_reader :entries
+
+        def initialize(entries)
+          @entries = entries
+        end
+
+        # Set key-value in the to-be-created correlation context
+        #
+        # @param [String] key The key to store this value under
+        # @param [String] value String value to be stored under key
+        def set_value(key, value)
+          @entries[key] = value.to_s
```

Comment:
Do we need the `.to_s` if we expect `value` to be a `String`?

Reply:
I was being extra safe, but I'm with you; we shouldn't enable poor API use.